frontend/src/component/feedbackNew/FeedbackComponent.tsx

```diff
@@ -349,50 +351,103 @@ export const FeedbackComponent = ({
                                         </ScoreHelpContainer>
                                     </StyledScoreContainer>
-                                    <Box>
-                                        <FormSubTitle>
-                                            {feedbackData.positiveLabel}
-                                        </FormSubTitle>
-                                        <TextField
-                                            placeholder='Your answer here'
-                                            style={{ width: '100%' }}
-                                            name='positive'
-                                            multiline
-                                            rows={3}
-                                            variant='outlined'
-                                            size='small'
-                                            InputLabelProps={{
-                                                style: {
-                                                    fontSize:
-                                                        theme.fontSizes
-                                                            .smallBody,
-                                                },
-                                            }}
-                                        />
-                                    </Box>
-                                    <Box>
-                                        <FormSubTitle>
-                                            {
-                                                feedbackData.areasForImprovementsLabel
-                                            }
-                                        </FormSubTitle>
-                                        <TextField
-                                            placeholder='Your answer here'
-                                            style={{ width: '100%' }}
-                                            multiline
-                                            name='areasForImprovement'
-                                            rows={3}
-                                            InputLabelProps={{
-                                                style: {
-                                                    fontSize:
-                                                        theme.fontSizes
-                                                            .smallBody,
-                                                },
-                                            }}
-                                            variant='outlined'
-                                            size='small'
-                                        />
-                                    </Box>
+
+                                    {feedbackComments !== false &&
+                                    feedbackComments.enabled &&
+                                    feedbackComments.name ===
+                                        'withoutTextfield' ? (
```

Comment:
why is field lower case? Can we also keep the variant names more business centric (the role the component plays) not UI centric so that it's not coupled to whether we use text field/text area/dropdown?

Reply:
I don't have strong feelings about this. I chose this name because in the context of the code it made perfect sense in order to explain what the condition does. What would prefer instead?

Reply:
maybe withComment and withoutComment variants